code/upaya/app/policies/tech_support_policy.rb

```diff
@@ -0,0 +1,17 @@
+class TechSupportPolicy < Struct.new(:user, :tech_support)
+  def index?
+    user.admin? || user.tech?
+  end
+
+  def search?
+    user.admin? || user.tech?
+  end
+
+  def show?
+    user.admin? || user.tech?
+  end
+
+  def reset?
+    user.admin? || user.tech?
+  end
+end
```

Comment:
I am assuming yes for now, esp. since we can copy/paste to show that functionality.